### PR TITLE
Replace `ping` with `ping_2`

### PR DESCRIPTION
**Pull request by tsoutsman**

### CODE REVIEW

File: applications/ping/src/lib.rs [hunk 1]
```diff
@@ -1,356 +1,184 @@
-//! This application pings a specific IPv4 address and gets ping statistics.
-//! Important: QEMU does not support the ICMP protocol by default so it's important to 
-//! run this command: sudo sh -c "echo \"0 2147483647\" > /proc/sys/net/ipv4/ping_group_range"
-//! in the environment prior to running this application
```
Comment (kevinaboos) [inline applications/ping/src/lib.rs:4]: it'd be nice to retain this info. Can you move it into the README (and/or the relevant book section about running on QEMU)?

**Review by kevinaboos — CHANGES_REQUESTED** [review]
Looks good, thanks! One small request before merging.

File: applications/ping/src/lib.rs (hunk 1 above)
Comment (tsoutsman) [reply applications/ping/src/lib.rs:4]: 👍 I actually don't think I never needed to run that command on fedora or macos but I'll add it anyway.

**Review by kevinaboos — APPROVED** [review]
Excellent, thanks!

**Pull request merged**